example/example.html

```diff
@@ -29,20 +29,34 @@
   <div id="banner"></div>
 
   <script type="text/javascript">
+    // Init Parallax
     var parallax = new Parallax();
     parallax.init("banner", "banner.jpg", "down");
 
     var el = document.getElementById("banner");
 
+    // Setup Parallax Functions
     function parallaxLoad() {
       el.style.backgroundPosition = ('0 '+parallax.scroll()+'px');
     }
-    window.addEventListener('scroll', function() {
+    function parallaxScroll() {
       el.style.backgroundPosition = ('0 '+parallax.scroll()+'px');
-    });
-    window.addEventListener('resize', function() {
+      requestAnimationFrame(parallaxScroll);
+    }
+    function parallaxResize() {
       el.style.backgroundPosition = ('0 '+parallax.resize()+'px');
-    });
+      setTimeout(function() { requestAnimationFrame(parallaxResize); }, 200); // 5fps

```

Comment:
This isn't going to do what you think it does. What will happen is that every time the resize event is triggered, it will execute this setTimeout, to run your function 200ms later. 
This means that if the resize event is triggered 50 times, your function will execute 50 times, delayed by 200ms from the first time the event was triggered. Make sense? 
What you want is to debounce, which will only execute your function once per time period, no matter how many times it is called within that time period.